src/frontend/js/components/DashBoard/Navigator/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { NavLink, useRoutes } from 'react-router-dom';
+import { ReactNode } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import useDashboardRoutes from '../hooks/useDashboardRoutes';
+import { Spinner } from '../../Spinner';
+
+export const messages = defineMessages({
+  loadingNavigator: {
+    id: 'components.Dashboard.Navigator.loadingInitial',
+    defaultMessage: 'Loading navigator',
+    description:
+      'Accessible text for the initial loading spinner displayed when navigator is fetching',
+  },
+});
+
+const DashBoardNavigator = () => {
+  const { intlDashboardRoutes } = useDashboardRoutes();
+  const routesNode = useRoutes(intlDashboardRoutes);
```

Comment:
I wonder if `useDashboardRoutes` should not be in charge to return those route nodes instead of the routes map object ?